utils/hooks.ts

```diff
@@ -109,12 +110,21 @@ const useGeocodeQuery = () => {
           setResults([]);
         } else {
           const filteredResults = filterDuplicatePlaces(nominatimResults);
-          const formattedResults = filteredResults.map((result) => ({
-            location: new LngLat(Number(result["lon"]), Number(result["lat"])),
-            name: result["display_name"],
-            simplifiedName: simplifyPlaceDisplayName(result),
-            isRegion: !nonRegionKeys.some((k) => k in result.address),
-          }));
+          const formattedResults = filteredResults.map((result) => {
+            
+            const firstElem:any = result["boundingbox"].shift();
+            const lastElem:any = result["boundingbox"].pop();
+            result["boundingbox"].push(firstElem);
+            result["boundingbox"].unshift(lastElem);
```

Comment:
How come you have to swap the first and last value? What would happen if you use the bounding box values given by the API as is?

Reply:
for some reason, the values obtained by the API & the map, seem to not match, these 4 values define the 'limits' of a box, and if I don't do this, this 'box'/'coordinates' point to a random place